acquia/acquia_cms
# ACMS-825: Added template for maintenance and update php page.

**Motivation**

Fixes #[ACMS-812](https://backlog.acquia.com/browse/ACMS-812)

**Proposed changes**

Provide brand new experience for acquia cms maintenance and update php page.

**Testing steps**

Install site check newly created install page for acquia cms distribution.
Install site, put site on maintenance mode and check newly created maintenance page for acquia cms distribution.
Visit /update.php to check newly created update.php page experience.

**Merge requirements**
- [x] Depends upon ACMS-812 PR for all changes. https://github.com/acquia/acquia_cms/pull/839
